code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -1,329 +0,0 @@
-	var/heal_amt = 2 * power
-
-	var/list/parts = M.get_damaged_bodyparts(1,0) //brute only
-
-	if(!parts.len)
-		return
-
-	for(var/obj/item/bodypart/L in parts)
-		if(L.heal_damage(heal_amt/parts.len, 0))
-			M.update_damage_overlays()
-
-	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#FF3333")
-
-	return 1
-
-
-/*
-//////////////////////////////////////
-
-Flesh Mending
-
-	No resistance change.
-	Decreases stage speed.
-	Decreases transmittablity.
-	Fatal Level.
-
-Bonus
-	Heals brute damage over time. Turns cloneloss into burn damage.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/heal/brute/plus
-
-	name = "Flesh Mending"
-	desc = "The virus rapidly mutates into body cells, effectively allowing it to quickly fix the host's wounds."
-	stealth = 0
-	resistance = 0
-	stage_speed = -2
-	transmittable = -2
-	level = 8
-
-/datum/symptom/heal/brute/plus/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/heal_amt = 4 * power
-
-	var/list/parts = M.get_damaged_bodyparts(1,0) //brute only
-
-	if(M.getCloneLoss() > 0)
-		M.adjustCloneLoss(-1)
-		M.take_bodypart_damage(0, 1) //Deals BURN damage, which is not cured by this symptom
-		if(!hide_healing)
-			new /obj/effect/temp_visual/heal(get_turf(M), "#33FFCC")
-
-	if(!parts.len)
-		return
-
-	for(var/obj/item/bodypart/L in parts)
-		if(L.heal_damage(heal_amt/parts.len, 0))
-			M.update_damage_overlays()
-
-	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#CC1100")
-
-	return 1
-
-/*
-//////////////////////////////////////
-
-Tissue Regrowth
-
-	Little bit hidden.
-	Lowers resistance tremendously.
-	Decreases stage speed tremendously.
-	Decreases transmittablity temrendously.
-	Fatal Level.
-
-Bonus
-	Heals burn damage slowly over time.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/heal/burn
-
-	name = "Tissue Regrowth"
-	desc = "The virus recycles dead and burnt tissues, speeding up the healing of damage caused by burns."
-	stealth = 1
-	resistance = -4
-	stage_speed = -4
-	transmittable = -4
-	level = 6
-
-/datum/symptom/heal/burn/Heal(mob/living/carbon/M, datum/disease/advance/A)
-	var/heal_amt = 2 * power
-
-	var/list/parts = M.get_damaged_bodyparts(0,1) //burn only
-
-	if(!parts.len)
-		return
-
-	for(var/obj/item/bodypart/L in parts)
-		if(L.heal_damage(0, heal_amt/parts.len))
-			M.update_damage_overlays()
-
-	if(prob(base_message_chance) && !hide_healing)
-		new /obj/effect/temp_visual/heal(get_turf(M), "#FF9933")
-	return 1
-
-
-/*
-//////////////////////////////////////
-
-Heat Resistance //Needs a better name
-
-	No resistance change.
-	Decreases stage speed.
-	Decreases transmittablity.
-	Fatal Level.
-
-Bonus
-	Heals burn damage over time, and helps stabilize body temperature.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/heal/burn/plus
-
-	name = "Temperature Adaptation"

```

Comment:
at least keep this

Reply:
actually, being able to walk through space is good for the game, since we don't already have a job that lets you do that.

Reply:
We have a alternativefor this its in genetics.

Reply:
actually, being able to walk through space is good for the game, since we don't already have a job that lets you do that.